src/components/ordertable.main.less

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+ .order-main-inner {
+  .order-table {
+    border: 1px solid #ddd;
+    margin-bottom: 0;
+    border-collapse: collapse;
+    border-spacing: 0;
+
+    & > tbody > tr:nth-child(odd) > td,
+    & > tbody > tr:nth-child(odd) > th {
+      background-color: #e9e9e9;
+    }
+
+    .table-headings {
+      background: @mainColor;
+      color: #fff;
+      font-weight: 400;
+      font-size: 16px;
+
+      .heading-quantity {
+        text-align: center;
+      }
+    }
+
+
+    @media screen and (max-width: @mobileWidth - 1px) {
+      display: block;
+      width: 100%;
+      
+      thead {
+        display: block;
+        width: 100%;
+
+        tr {
+          position: absolute;
+          top: -9999px;
+          left: -9999px;
+        }
+      }
+
+      tbody {
+        display: block;
+        width: 100%;
+
+        tr {
+          display: block;
+          width: 100%;
+          border: 1px solid #eee;
+          margin-bottom: 30px;
+
+          td {
+            display: block!important;
```

Comment:
Use `!important` only when overriding default Bootstrap styles. If you have to use it put a space before it.